### PR TITLE
[ES] Add `G` missing stubs (2)

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing G stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish.

28 stubs in total, divided into 4 pull requests of 7 stubs each. Continued in: #10626

~~Let me know if there are any problems with the image because of its _Christmas_ design~~ **I added the image without the christmas style**

### CODE REVIEW

File: wiki/Gameplay/Input_device/Beatmania_controller/es.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+![](img/beatmania-controller.jpg "Mando de Beatmania de 7 botones")
+
+El **mando de Beatmania** es un tipo de mando de juego que es compatible con la serie de juegos *[Beatmania](https://es.wikipedia.org/wiki/Beatmania_(serie))*.
+
+Debido a su naturaleza, los mandos de Beatmania se usan con mayor frecuencia como dispositivos de entrada en [osu!mania](/wiki/Game_mode/osu!mania).
+
+## Métodos de entrada
+
+Los jugadores que tengan un mando de *Beatmania* de Playstation 2 (PS2), pueden usar un adaptador para conectar el mando a su computadora mediante USB y asignar las teclas en las opciones. Sin embargo, los jugadores que tienen un mando *DJ DAO IIDX*, pueden simplemente conectar el mando a su computadora mediante USB y asignar las teclas.
```
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Beatmania_controller/es.md:15]:
```suggestion
Los jugadores que tengan un mando de *Beatmania* de Playstation 2 (PS2), pueden usar un adaptador para conectar el mando a su ordenador mediante USB y asignar las teclas en las opciones. Sin embargo, los jugadores que tienen un mando *DJ DAO IIDX*, pueden simplemente conectar el mando a su ordenador mediante USB y asignar las teclas.
```

File: wiki/Gameplay/Input_device/Dance_pad/es.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+stub: true
+---
+
+# Pad de baile
+
+![](img/dance-pad.jpg "Un pad de baile de 4 botones")
+
+Un **pad de baile**, o **tapete de baile**, es un tipo de mando para juegos de ritmo que usa la ubicación de los pies del jugador como entrada. Como se muestra a continuación, los pads de baile se colocan en el suelo y permiten al usuario moverse entre cuatro y nueve «direcciones» para activar las teclas. Este tipo de mandos se usan con mayor frecuencia en juegos de ritmo como *[Dance Dance Revolution](https://es.wikipedia.org/wiki/Dance_Dance_Revolution_(serie))*.
+
+El tapete de baile de [Wii](https://es.wikipedia.org/wiki/Wii) requiere el uso de un adaptador de mando [GameCube](https://es.wikipedia.org/wiki/Nintendo_GameCube) para funcionar con una computadora. Una vez que hayas conectado el pad de baile a tu computadora, puedes asignar las teclas correctas en los ajustes de osu!.
```
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Dance_pad/es.md:11]: pad o tapete?
```suggestion
El pad de baile de [Wii](https://es.wikipedia.org/wiki/Wii) requiere el uso de un adaptador de mando [GameCube](https://es.wikipedia.org/wiki/Nintendo_GameCube) para funcionar con un ordenador. Una vez que hayas conectado el pad de baile a tu ordenador, puedes asignar las teclas correctas en los ajustes de osu!.
```

File: wiki/Gameplay/Input_device/Drum_controller/es.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# Mando de tambor
+
+![](img/taiko-drum.jpg "Tambor de Taiko")
+
+Los **mandos de tambor** son dispositivos de entrada diseñados para funcionar con juegos de tambor tipo Taiko. Por lo general, tienen cuatro áreas de entrada: centro derecho del tambor, centro izquierdo del tambor, borde derecho del tambor y borde izquierdo del tambor, y cada entrada (tecla) se activa con una baqueta que se golpea en el área correspondiente del tambor.
+
+## Entrada
+
+Los propietarios de los mandos de tambor de Taiko para [PlayStation 4 (PS4)](https://es.wikipedia.org/wiki/PlayStation_4) o [Nintendo Switch](https://es.wikipedia.org/wiki/Nintendo_Switch) tendrán que conectar sus mandos a la computadora mediante USB y asignar las teclas correctas en los ajustes de osu!. Los propietarios del mando Tatacon de [Wii](https://es.wikipedia.org/wiki/Wii) deberán emparejar su mando con su computadora antes de habilitar la opción de compatibilidad con Wiimote/Tatacon en los ajustes de osu!. La mayoría de los mandos de tambor de terceros también funcionan con osu!, pero la compatibilidad puede variar.
```
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Drum_controller/es.md:13]:
```suggestion
Los propietarios de los mandos de tambor de Taiko para [PlayStation 4 (PS4)](https://es.wikipedia.org/wiki/PlayStation_4) o [Nintendo Switch](https://es.wikipedia.org/wiki/Nintendo_Switch) tendrán que conectar sus mandos a la computadora mediante USB y asignar las teclas correctas en los ajustes de osu!. Los propietarios del mando Tatacon de [Wii](https://es.wikipedia.org/wiki/Wii) deberán emparejar su mando con su computadora antes de habilitar la opción de compatibilidad con Wiimote/Tatacon en los ajustes de osu!. La mayoría de los mandos de tambor de terceros también funcionan con osu!, pero la compatibilidad puede variar.
```
```suggestion
Los propietarios de los mandos de tambor de Taiko para [PlayStation 4 (PS4)](https://es.wikipedia.org/wiki/PlayStation_4) o [Nintendo Switch](https://es.wikipedia.org/wiki/Nintendo_Switch) tendrán que conectar sus mandos a la computadora mediante USB y asignar las teclas correctas en los ajustes de osu!. Los propietarios del mando Tatacon de [Wii](https://es.wikipedia.org/wiki/Wii) deberán emparejar su mando con su ordenador antes de habilitar la opción de compatibilidad con Wiimote/Tatacon en los ajustes de osu!. La mayoría de los mandos de tambor de terceros también funcionan con osu!, pero la compatibilidad puede variar.
```

Comment (agatemosu) [inline wiki/Gameplay/Input_device/Drum_controller/es.md:13]:
```suggestion
Los propietarios de los mandos de tambor de Taiko para [PlayStation 4 (PS4)](https://es.wikipedia.org/wiki/PlayStation_4) o [Nintendo Switch](https://es.wikipedia.org/wiki/Nintendo_Switch) tendrán que conectar sus mandos a la computadora mediante USB y asignar las teclas correctas en los ajustes de osu!. Los propietarios del mando Tatacon de [Wii](https://es.wikipedia.org/wiki/Wii) deberán emparejar su mando con su ordenador antes de habilitar la opción de compatibilidad con Wiimote/Tatacon en los ajustes de osu!. La mayoría de los mandos de tambor de terceros también funcionan con osu!, pero la compatibilidad puede variar.
```

File: wiki/Gameplay/Input_device/Mouse/es.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+stub: true
+---
+
+# Ratón
+
+El **ratón** es una forma muy popular de jugar osu!. Puedes utilizar el ratón para apuntar y hacer clic. El botón central del ratón está configurado para pausar el juego (se puede desactivar en las opciones) y la rueda de desplazamiento se puede usar para cambiar el volumen.
```
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Mouse/es.md:7]:
```suggestion
El **ratón** es una forma muy popular de jugar osu!. Puedes usar el ratón para apuntar y hacer clic. El botón central del ratón está configurado para pausar el juego (se puede desactivar en las opciones) y la rueda de desplazamiento se puede usar para cambiar el volumen.
```

File: wiki/Gameplay/Input_device/Beatmania_controller/es.md (hunk 1 above)
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Beatmania_controller/es.md:15]:
```suggestion
Los jugadores que tengan un mando de *Beatmania* de Playstation 2 (PS2), pueden usar un adaptador para conectar el mando a su ordenador mediante USB y asignar las teclas en las opciones. Sin embargo, los jugadores que tienen un mando *DJ DAO IIDX*, pueden simplemente conectar el mando a su ordenador mediante USB y asignar las teclas.
```

File: wiki/Gameplay/Input_device/Dance_pad/es.md (hunk 2 above)
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Dance_pad/es.md:11]:
```suggestion
El pad de baile de [Wii](https://es.wikipedia.org/wiki/Wii) requiere el uso de un adaptador de mando [GameCube](https://es.wikipedia.org/wiki/Nintendo_GameCube) para funcionar con un ordenador. Una vez que hayas conectado el pad de baile a tu ordenador, puedes asignar las teclas correctas en los ajustes de osu!.
```

File: wiki/Gameplay/Input_device/Drum_controller/es.md (hunk 3 above)
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Drum_controller/es.md:13]:
```suggestion
Los propietarios de los mandos de tambor de Taiko para [PlayStation 4 (PS4)](https://es.wikipedia.org/wiki/PlayStation_4) o [Nintendo Switch](https://es.wikipedia.org/wiki/Nintendo_Switch) tendrán que conectar sus mandos a la computadora mediante USB y asignar las teclas correctas en los ajustes de osu!. Los propietarios del mando Tatacon de [Wii](https://es.wikipedia.org/wiki/Wii) deberán emparejar su mando con su ordenador antes de habilitar la opción de compatibilidad con Wiimote/Tatacon en los ajustes de osu!. La mayoría de los mandos de tambor de terceros también funcionan con osu!, pero la compatibilidad puede variar.
```

File: wiki/Gameplay/Input_device/Mouse/es.md (hunk 4 above)
Comment (agatemosu) [inline wiki/Gameplay/Input_device/Mouse/es.md:7]:
```suggestion
El **ratón** es una forma muy popular de jugar osu!. Puedes usar el ratón para apuntar y hacer clic. El botón central del ratón está configurado para pausar el juego (se puede desactivar en las opciones) y la rueda de desplazamiento se puede usar para cambiar el volumen.
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**